### PR TITLE
PE-2410: folder and multiple files and cannot be uploaded using ar connect

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/1ns91i56h6vf8
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/6j7jelopf2250

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ dependencies:
   arweave:
     git:
       url: https://github.com/ardriveapp/arweave-dart
-      ref: v3.3.0
+      ref: PE-2410-folder-and-multiple-files-and-cannot-be-uploaded-using-ar-connect
```
Comment (matibat) [inline pubspec.yaml:38]: Reminder to update this when the Arweave-dart gets released

File: lib/blocs/upload/upload_handles/bundle_upload_handle.dart [hunk 2]
```diff
@@ -50,21 +51,37 @@ class BundleUploadHandle implements UploadHandle {
     required Wallet wallet,
   }) async {
     final bundle = await DataBundle.fromHandles(
+      parallelize: false,
```
Comment (matibat) [inline lib/blocs/upload/upload_handles/bundle_upload_handle.dart:54]: I'm not sure about this line. I'll see if I find out when reviewing Arweave-Dart

**Review by matibat — COMMENTED** [review]
Looks good -

File: lib/blocs/upload/upload_handles/bundle_upload_handle.dart (hunk 2 above)
Comment (matibat) [reply lib/blocs/upload/upload_handles/bundle_upload_handle.dart:54]: If running this method synchronously is the fix and this affects ArConnect only, then we could do something like...
```suggestion
      parallelize: !isArConnect,
```

Comment (thiagocarvalhodev) [reply lib/blocs/upload/upload_handles/bundle_upload_handle.dart]: That makes total sense. Thanks for this good catch!

Comment (thiagocarvalhodev) [reply lib/blocs/upload/upload_handles/bundle_upload_handle.dart]: [fc40ea6](https://github.com/ardriveapp/ardrive-web/pull/742/commits/fc40ea60ee52bb7cccabbfd548f10234e43ae643)

**Review by karlprieb — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Pull request merged**